cartographer/mapping_3d/sparse_pose_graph.cc

```diff
@@ -386,6 +386,10 @@ void SparsePoseGraph::WaitForAllComputations() {
   locker.Await([&notification]() { return notification; });
 }
 
+void SparsePoseGraph::FinishTrajectory(const int trajectory_id) {
+  LOG(FATAL) << "Not yet implemented for 3D.";
```

Comment:
This would break 3D localization which we now support. Change to a `TODO` comment similar to 2D?